src/cli/domain/watch.js

```diff
@@ -1,33 +1,29 @@
 'use strict';
 
 const path = require('path');
-const watch = require('watch');
+const chokidar = require('chokidar');
 
 const settings = require('../../resources/settings');
 
 module.exports = function(dirs, baseDir, changed) {
-  try {
-    watch.watchTree(
-      path.resolve(baseDir),
-      {
-        interval: 0.5,
-        ignoreUnreadableDir: true,
-        ignoreDotFiles: false,
-        filter: fileOrDir =>
-          settings.filesToIgnoreOnDevWatch.test(fileOrDir) === false
-      },
-      (fileName, currentStat, previousStat) => {
-        if (!!currentStat || !!previousStat) {
-          const componentDir = dirs.find(dir =>
-            Boolean(fileName.match(escapeRegularExpression(dir + path.sep)))
-          );
-          changed(null, fileName, componentDir);
-        }
-      }
+  const watcher = chokidar.watch(path.resolve(baseDir), {
```

Comment:
is it possible for this code to throw? or was it just expected syntax before with `watch`?

Reply:
No, chokidar uses event emitter, so errors will go to the error event.